tests/test_bgp_af_aa.rb

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2017 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/bgp_af'
+require_relative '../lib/cisco_node_utils/bgp_af_aggr_addr'
+
+include Cisco
+# TestBgpAfAggrAddr - Minitest for BgpAFAggrAddr class
+class TestBgpAfAggrAddr < CiscoTestCase
+  @@pre_clean_needed = true # rubocop:disable Style/ClassVars
```

Comment:
Do you need to skip any platforms for this or is it supported on n3|7|9k platforms?